packages/sdk/test/cross-chain-provider.spec.ts

```diff
@@ -783,46 +760,204 @@ describe('CrossChainProvider', () => {
   })
 
   describe('getMessageStatus', () => {
+    let scc: Contract
+    let l1Messenger: Contract
+    let l2Messenger: Contract
+    let provider: CrossChainProvider
+    beforeEach(async () => {
+      scc = (await (await ethers.getContractFactory('MockSCC')).deploy()) as any
```

Comment:
nit: nested `await`s defeat the purpose of having `await` - making async code very readable

Reply:
I did this in other places, I'll make a comment with a TODO and turn this into a good first issue for someone to get rid of these nested awaits everywhere within this package.